assets/src/story-embed-block/block/index.js

```diff
@@ -18,9 +18,10 @@
  * WordPress dependencies
  */
 import { __ } from '@wordpress/i18n';
-// TODO: Replace with web stories icon.
-import { page as icon } from '@wordpress/icons';
-
+/**
+ * Internal dependencies
```

Comment:
No there are two `Internal dependencies` comments

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/2744/commits/5fc012f143d07c19823157d242867f3befaecb22